installer/manifests/keptn/templates/api-gateway-nginx.yaml

```diff
@@ -90,6 +90,10 @@ data:
       proxy_busy_buffers_size 256k;
     }
 
+    location @error401 {
+      return 302 /api/v1/auth;
```

Comment:
I don't think this would work since you create a loop: user request -> (auth endpoint -> failure)+
Besides, why 302?

Reply:
works, but will try the solution